src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -0,0 +1,182 @@
+import { Platform } from "react-native";
+
+const script = (nightMode, position) => {
+  const listener = Platform.OS === "android" ? "document" : "window";
+  return `
+
+var autoScrollTimeout;
+var autoScrollSpeed = 0;
+var scrollMultiplier = 1.0;
+var dragging = false;
+var holding = false;
+var holdTimer;
+var curPosition = 0;
+var isScrolling;
+var isManuallyScrolling = false;
```

Comment:
Can we use `let` or `const` here instead of `var` variables in this entire file?

Reply:
fixed